src/service/api.tsx

```diff
@@ -93,9 +107,10 @@ const api = {
   getDataByIds(ids: number[]) {
     const urls = ids.map((id: number) => `movie/${id}?${API_KEY}`);
     const requests = urls.map(
-      async (url: any) => await axios.get(url).then((res: { data: any; }) => res.data)
+      async (url: any) =>
```

Comment:
Why "any"? does url have string type?

Reply:
Is it possible to describe type for "data" instead of any?

Reply:
thanks for pointing it out. Changed types for 'url' and 'data'